Sources/Covfefe/Terminal.swift

```diff
@@ -132,9 +144,14 @@ extension Terminal: Codable {
 		case .regularExpression:
 			let pattern = try container.decode(String.self, forKey: .value)
 			self = try .regularExpression(expression: NSRegularExpression(pattern: pattern, options: []), hash: pattern.hashValue)
+		
+		case .characterSet:
+			let aSet = try container.decode(CharacterSet.self, forKey: .value)
+			self = .characterSet(set: aSet, hash: aSet.hashValue)
 		}
 	}
 	
+	#warning("Codable is synthetisis is available, should we remove this?")
```

Comment:
Yep, can be removed.

Reply:
It would work nicely. However, there is an issue
```initializer requirement 'init(from:)' can only be satisfied by a 'required' initializer in the definition of non-final class 'NSRegularExpression ```

The issue could be solved by introducing a box for the NSRegularExpression, but doing so would defeat the original purpose of hamoving the manual conformance: removing the manual conformace.

Therefore I would defer this change to a later date.